TECHSPEC.md

```diff
@@ -27,16 +27,18 @@ A Command Line Application for printing receipts to Terminal
     <figcaption>Layout Mockup in Indesign </figcaption>
 </figure>
 
-<figure>
-    <img src="./images/layout_rev1.jpeg" alt="drawing" width="400"/>
-    <figcaption>Revised with new logo requirements & structure</figcaption>
-</figure>
-
-<figure>
-    <img src="./images/HTMLelements.png" alt="drawing" width="400"/>
-    <figcaption>Planning HTML elements </figcaption>
-</figure>
-
+<table>
```

Comment:
This is an A+ layout. I'd like to see it very near the top of the README as it showcases your design skills & immediately shows what the project is capable of.